teku/src/main/java/tech/pegasys/teku/cli/slashingprotection/ImportCommand.java

```diff
@@ -65,11 +67,12 @@ public void run() {
     try {
       SUB_COMMAND_LOG.display("Reading slashing protection data from: " + importFile);
       final Optional<String> errorCondition = importer.initialise(importFile);
-      errorCondition.ifPresent(s -> SUB_COMMAND_LOG.exit(1, s));
+      errorCondition.ifPresent(s -> SUB_COMMAND_LOG.exit(FATAL_EXIT_CODE, s));
     } catch (IOException e) {
       String cause = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
       SUB_COMMAND_LOG.exit(
-          1, String.format("Failed to read from import file: %s. %s", importFile, cause));
+          FATAL_EXIT_CODE,
+          String.format("Failed to read from import file: %s. %s", importFile, cause));
```

Comment:
i'd probalby argue the same for all the tools, i'd leave them at a 1 unless it truly is a usage issue. maybe an exit code for usage if we ever actually do have that scenario...